ontohgis.ttl

```diff
@@ -6537,7 +6537,7 @@ ontohgis:administrative_type_1 rdf:type owl:Class ;
                                rdfs:subClassOf ontohgis:administrative_type_24 ;
                                rdfs:comment """\"1808: Städte-Ordnung.
 Gminy miejskie i gminy wiejskie były jednostkami organizacji samorządowej (komunalnej), ale równocześnie pełniły funkcje podstawowych jednostek administracji terytorialnej. M. Bär, s. 228: „Die Landgemeinde ist außer einem Kommunalverband auch ein Bezirk der allgemeinen Landesverwaltung für die mannigfachen Geschäfte des staatlichen Polizei- und Finanzwesens, insbesondere für die Handhabung der Ortspolizei, und der Gemeindevorsteher ist das Organ des Amtsvorstehers für die örtliche Polizeiverwaltung“. W. Roger W.Frotscher, „Überblick über die Verwaltungsorganisation in den Bundesstaaten, w: Deutsche Verwaltungsgeschichte, Bd. 3 hrsg. von K.G.A. Jeserich, H. Pohl und G. Ch. v. Unruh, Stuttgart 1984, S. 417: „Der preußische Staat endete beim Landrat. In diesem oft zitierten Satz kommt zum Ausdruck, dass Gemeindeorgane, auch soweit sie staatliche Angelegenheiten wahrnahmen, nicht als staatliche, sondern als kommunale Behörden tätig wurden. Sie handelten, etwa als Ortspolizei `für den Staat` in einem ihnen übertragenen Wirkungskreis.\""""@de ;
-                               rdfs:isDefinedBy ontohgis:administrative_system_4 ;
+                               rdfs:isDefinedByIRI ontohgis:administrative_system_4 ;
```

Comment:
```suggestion
                               ontohgis:isDefinedByIRI ontohgis:administrative_system_4 ;
```

The same applies to all instances of _rdfs:isDefinedByIRI_ and _rdfs:isDefinedByLiteral_, i.e. perform the following substitutions:

_rdfs:isDefinedByIRI_ -> _**ontohgis**:isDefinedByIRI_
_rdfs:isDefinedByLiteral_ -> _**ontohgis**:isDefinedByLiteral_

with the following command:
```bash
sed -i -e 's/\(\s\)\S\+:isDefinedByIRI/\1ontohgis:isDefinedByIRI/gi' \
       -e 's/\(\s\)\S\+:isDefinedByLiteral/\1ontohgis:isDefinedByLiteral/gi' ontohgis.ttl
```

Reply:
@mereolog can you have a look?

Reply:
> @mereolog can you have a look?

I did - see: https://github.com/ontohgis/ontology/pull/14#pullrequestreview-1418508728.

Reply:
Thanks, @mereolog  but I do not understand... Simply, I need a decision do we can merge my changes or not.

Reply:
> The same applies to all instances of _rdfs:isDefinedByIRI_ and _rdfs:isDefinedByLiteral_, i.e. perform the following substitutions:
> 
> _rdfs:isDefinedByIRI_ -> _**ontohgis**:isDefinedByIRI_ _rdfs:isDefinedByLiteral_ -> _**ontohgis**:isDefinedByLiteral_
> 
> with the following command:
> 
> ```shell
> sed -i -e 's/\(\s\)\S\+:isDefinedByIRI/\1ontohgis:isDefinedByIRI/gi' \
>        -e 's/\(\s\)\S\+:isDefinedByLiteral/\1ontohgis:isDefinedByLiteral/gi' ontohgis.ttl
> ```

I agree with these comments.

Reply:
@mereolog great, thanks. 
@przemekgradzki Please have a look once again at performed changes I already pushed.